### PR TITLE
Add MPC wallet for onramp and offramp test

### PR DESCRIPTION
**Pull request by wen-zhao-cb**
Added a MPC wallet generation for onramp and offramp test

- Create or load existing wallet based on network 
  - Wallet seed will be stored in local file for simplicity
  - Show wallet address and wallet balance

- Create onramp url based on wallet
- Create offramp url based on wallet
- Fetch sell offramp transaction
- Send crypto for offramp

https://github.com/user-attachments/assets/b39f482d-1b39-47cb-a552-691959c8dc4b

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: app/components/TransferCrypto.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const [transfer_result, setTransferResult] = useState("");
+
+  useEffect(() => {
+    const getOfframpTransaction = async () => {
+      const transaction = await getSellTransactionStatus({
+        partner_user_id: user_id,
+      });
+      try {
+        if (transaction) {
+          console.log(transaction);
```
Comment (steveviselli-cb) [inline app/components/TransferCrypto.tsx:22]: @wen-zhao-cb I think we should either remove the console logs, or give them a bit more context i.e. `console.log('[DEBUG] Got offramp transaction:', transaction);`

File: app/components/TransferCrypto.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          console.log(transaction.asset);
+          setAmount(+transaction.sell_amount.value);
```
Comment (steveviselli-cb) [inline app/components/TransferCrypto.tsx:24]: @wen-zhao-cb does the sell amount come back as a negative number?

File: app/components/WalletBox.tsx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    setOnrampLink(
+      `https://pay.coinbase.com/buy/select-asset?appId=${project_id}` +
+        `&addresses={"${address}":["${shorten_network}"]}`
+    );
+  }, [address, network]);
+
+  const generateOfframpLink = useCallback(async () => {
+    let shorten_network = network.split("-")[0];
+    setOfframpLink(
+      `https://pay.coinbase.com/v3/offramp/input?appId=${project_id}` +
```
Comment (steveviselli-cb) [inline app/components/WalletBox.tsx:53]: @wen-zhao-cb is this really not /v3/sell? Can we change that before release?

**Review by steveviselli-cb — APPROVED** [review]
Awesome stuff! Couple minor comments we can address later if you want.

File: app/components/WalletBox.tsx (hunk 3 above)
Comment (wen-zhao-cb) [reply app/components/WalletBox.tsx:53]: It's already changed, I'll update the url

File: app/components/TransferCrypto.tsx (hunk 1 above)
Comment (wen-zhao-cb) [reply app/components/TransferCrypto.tsx:22]: I've cleaned up few console logs.

File: app/components/TransferCrypto.tsx (hunk 2 above)
Comment (wen-zhao-cb) [reply app/components/TransferCrypto.tsx:24]: no the value will never be negative. +transaction.sell_amount.value will just convert string to number

**Review by steveviselli-cb — APPROVED** [review]

**Pull request merged**